online/src/main/scala/ai/chronon/online/MetadataEndPoint.scala

```diff
@@ -15,7 +15,7 @@ object MetadataEndPoint {
   @transient implicit lazy val logger = LoggerFactory.getLogger(getClass)
 
   val ConfByKeyEndPointName = "ZIPLINE_METADATA"
-  val NameByTeamEndPointName = "CHRONON_METADATA_BY_TEAM"
+  val NameByTeamEndPointName = "CHRONON_ENTITY_BY_TEAM"
```

Comment:
do we have to hard code it ?

Reply:
@pengyu-hou Yep we have to. This is the data name we set on Mussel.